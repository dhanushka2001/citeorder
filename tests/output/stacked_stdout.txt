Output written to tests/stacked-fixed.md
