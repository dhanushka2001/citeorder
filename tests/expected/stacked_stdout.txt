Output written to stacked-fixed.md
